src/NServiceBus.MessagingBridge/Configuration/BridgeConfiguration.cs

```diff
@@ -54,7 +54,12 @@ internal FinalizedBridgeConfiguration FinalizeConfiguration(ILogger<BridgeConfig
             if (tranportsWithNoEndpoints.Any())
             {
                 var endpointNames = string.Join(", ", tranportsWithNoEndpoints);
-                throw new InvalidOperationException($"At least one endpoint needs to be configured for transport(s): {endpointNames}");
+                logger.LogWarning($"The following transport(s) have no endpoints: {endpointNames}");
```

Comment:
Logging should use structured logging to keep log event ID identical for different values of `endpointNames`:
```suggestion
                logger.LogWarning("The following transport(s) have no endpoints: {endpointNames}", endpointNames);
```